packages/jest-jasmine2/src/jasmine/Spec.js

```diff
@@ -121,10 +127,26 @@ Spec.prototype.onException = function onException(e) {
     return;
   }
 
+  const {expected, actual} = e;
+  let message;
+  if (expected && actual) {
+    const diffString = diff(expected, actual, {
+      expand: this.expand,
+    });
+    message = matcherHint('.toBe') +
+      '\n\n' +
+      `Expected value to be (using ===):\n` +
+      `  ${printExpected(expected)}\n` +
+      `Received:\n` +
+      `  ${printReceived(actual)}` +
+      (diffString ? `\n\nDifference:\n\n${diffString}` : '');
+  }
+
   this.addExpectationResult(
     false,
     {
-      matcherName: '',
+      matcherName: 'blah blah',
```

Comment:
If I leave `matcherName` as an empty string then what I get is the same thing as if I had not made any changes at all. I'm unsure of what this should be set to though.